governance/delegates/0x45127ec92b58c3a89e89f63553073adcaf2f1f5f/metrics.md

```diff
@@ -3,4 +3,5 @@ combined_participation: No Data
 poll_participation: No Data
 exec_participation: No Data
 communication: No Data
+start_date: 2030-01-01T00:00:00.000Z
```

Comment:
@LongForWisdom safe to assume this is a test of how a start date set in the future is handled and not a typo?

Reply:
Yep, Monet hasn't started yet technically (not til he's had his 'meet your delegate' meeting.) And yes, I was using 2030 as the 'Haven't set yet' value.